epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/output.py

```diff
@@ -251,17 +268,15 @@ def add_madevent_plugin_fct(self):
         which contains a series of functions and one dictionary variable TO_OVERWRITE
         that will be used to have temporary overwrite of all the key variable passed as string by their value.
         all variable that are file related should be called as madgraph.dir.file.variable
-        """
-        
+        """        
         plugin_path = os.path.dirname(os.path.realpath( __file__ ))
-        files.cp(pjoin(plugin_path, 'plugin_interface.py'), pjoin(self.dir_path, 'bin', 'internal'))
+        ###files.cp(pjoin(plugin_path, 'plugin_interface.py'), pjoin(self.dir_path, 'bin', 'internal')) # AV FIXME (added by OM, but file is missing?)
         files.cp(pjoin(plugin_path, 'launch_plugin.py'), pjoin(self.dir_path, 'bin', 'internal'))
```

Comment:
Ok will take a look (maybe we do not need this anymore (or I forget to push that file)

Reply:
Please do not resolve, stuff not yet resolve.
So I believe that this was for an old attempt to modify the run_card.dat (the attempt I did during the hackaton).
So the best is to fully remove that line. Thanks for forcing me to look at this line.